circonusllhist.go

```diff
@@ -372,14 +368,20 @@ func Deserialize(in io.Reader) (h *Histogram, err error) {
 }
 
 func (h *Histogram) Serialize(w io.Writer) error {
+	var bvs []*bin
+	for i := range h.bvs {
```

Comment:
I believe that filtering the list of bins like this is valid, since they should remain in order (in my testing, empty bins are simply the suffix to the `bvs` slice). Is this appropriate?

Reply:
If the purpose of the new list is to count them should should able to accomplish this without a second array to which you must append.  Do one range scan counting non-zero bins and then add the `bv.count != 0` as a predicate to the writeBin

Reply:
Done, updated :)